content/tracing/setup/cpp.md

```diff
@@ -0,0 +1,163 @@
+../scripts/install_dependencies.sh
+cmake ..
+make
+make install
+```
+
+Include `<datadog/opentracing.h>` and create the tracer:
+
+```cpp
+// tracer_example.cpp
+#include <datadog/opentracing.h>
+#include <iostream>
+#include <string>
+
+int main(int argc, char* argv[]) {
+  datadog::opentracing::TracerOptions tracer_options{"localhost", 8126, "compiled-in example"};
+  auto tracer = datadog::opentracing::makeTracer(tracer_options);
+
+  // Create some spans.
+  {
+    auto span_a = tracer->StartSpan("A");
+    span_a->SetTag("tag", 123);
+    auto span_b = tracer->StartSpan("B", {opentracing::ChildOf(&span_a->context())});
+    span_b->SetTag("tag", "value");
+  }
+
+  tracer->Close();
+  return 0;
+}
+```
+
+Just link against libdd_opentracing and libopentracing (making sure that they are both in your LD_LIBRARY_PATH):
+
+```bash
+g++ -o tracer_example tracer_example.cpp -ldd_opentracing -lopentracing
+./tracer_example
+```
+
+### Dynamic Loading
+
+```bash
+# Download and install OpenTracing-cpp
+wget https://github.com/opentracing/opentracing-cpp/archive/v1.5.0.tar.gz -O opentracing-cpp.tar.gz
+tar zxvf opentracing-cpp.tar.gz
+mkdir opentracing-cpp-1.5.0/.build
+cd opentracing-cpp-1.5.0/.build
+cmake ..
+make
+make install
+# Install dd-opentracing-cpp shared plugin.
+wget https://github.com/DataDog/dd-opentracing-cpp/releases/download/v0.3.7/linux-amd64-libdd_opentracing_plugin.so.gz
+gunzip linux-amd64-libdd_opentracing_plugin.so.gz -c > /usr/local/lib/libdd_opentracing_plugin.so
+```
+
+Include `<opentracing/dynamic_load.h>` and load the tracer from libdd_opentracing_plugin.so:
+
+```cpp
+// tracer_example.cpp
+#include <opentracing/dynamic_load.h>
+#include <iostream>
+#include <string>
+
+int main(int argc, char* argv[]) {
+  // Load the tracer library.
+  std::string error_message;
+  auto handle_maybe = opentracing::DynamicallyLoadTracingLibrary(
+      "/usr/local/lib/libdd_opentracing_plugin.so", error_message);
+  if (!handle_maybe) {
+    std::cerr << "Failed to load tracer library " << error_message << "\n";
+    return false;
+  }
+
+  // Read in the tracer's configuration.
+  std::string tracer_config = R"({
+      "service": "dynamic-load example",
+      "agent_host": "localhost",
+      "agent_port": 8126
+    })";
+
+  // Construct a tracer.
+  auto& tracer_factory = handle_maybe->tracer_factory();
+  auto tracer_maybe = tracer_factory.MakeTracer(tracer_config.c_str(), error_message);
+  if (!tracer_maybe) {
+    std::cerr << "Failed to create tracer " << error_message << "\n";
+    return false;
+  }
+  auto& tracer = *tracer_maybe;
+
+  // Create some spans.
+  {
+    auto span_a = tracer->StartSpan("A");
+    span_a->SetTag("tag", 123);
+    auto span_b = tracer->StartSpan("B", {opentracing::ChildOf(&span_a->context())});
+    span_b->SetTag("tag", "value");
+  }
+
+  tracer->Close();
+  return 0;
+}
+```
+
+Just link against libopentracing (making sure that libopentracing.so is in your LD_LIBRARY_PATH):
```

Comment:
```suggestion
Just link against `libopentracing`, making sure that `libopentracing.so` is in your `LD_LIBRARY_PATH`:
```

Reply:
Committing your suggestion, but would you also like to remove the "Just"? :D